--- a/configure
+++ b/configure
@@ -374,21 +374,6 @@
 fi
 if [ -n "$DEBUG" -a "$DEBUG" != no -a "$DEBUG" != false ]; then
 	echo "Adding debugging CFLAGS"
-	printf "Testing compiler supports address sanitisation ..."
-	cat <<EOF >_test.c
-int main(void) {
-	return 0;
-}
-EOF
-	if $CC -fsanitize=address _test.c -o _test 2>&3; then
-		echo "yes"
-		echo "# Compiler supports address sanitisation" >>$CONFIG_MK
-		echo "CFLAGS+=	-fsanitize=address" >>$CONFIG_MK
-		echo "LDFLAGS+=	-fsanitize=address" >>$CONFIG_MK
-	else
-		echo "no"
-	fi
-	rm -rf _test.c _test
 
 	cat <<EOF >>$CONFIG_MK
 CFLAGS+=	-g -Wall -Wextra
